components/OutivityDetail/index.js

```diff
@@ -42,4 +57,21 @@ const StyledCountryName = styled.span`
 const StyledOutivityImage = styled(Image)`
   overflow: clip;
   overflow-clip-margin: content-box;
+  height: auto;
+`;
+
+const StyledDeleteButton = styled.button`
```

Comment:
If you have multiple Styles with the same Usage (e.g. Buttons, Images, ...) consider creating a shared component in the future, so you have the same styling across the board.

Reply:
Thank you very much, that's a  very good suggestion, we'll work on that in the future 🤓 :)